x-pack/plugins/cases/server/saved_object_types/user_actions.ts

```diff
@@ -38,15 +35,25 @@ export const caseUserActionSavedObjectType: SavedObjectsType = {
           },
         },
       },
-      new_value: {
-        type: 'text',
-      },
-      old_value: {
-        type: 'text',
+      payload: {
+        dynamic: false,
+        properties: {
+          connector: {
+            dynamic: false,
```

Comment:
I don't think we need this here as it will inherit it from the parent: https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic.html

Reply:
Just waned to bring this up again since we didn't remove the inner `dynamic: false`. Unless we figured out that we do need it?